relayer/chain/substrate/writer.go

```diff
@@ -149,38 +161,25 @@ func (wr *Writer) write(_ context.Context, c types.Call) error {
 	return nil
 }
 
-// WriteMessage submits a "Bridge.submit" call
-func (wr *Writer) WriteMessage(ctx context.Context, msg *chain.Message) error {
-	c, err := types.NewCall(&wr.conn.metadata, "Bridge.submit", msg.AppID, msg.Payload)
-	if err != nil {
-		return err
-	}
-
-	err = wr.write(ctx, c)
-	if err != nil {
-		return err
-	}
+// WriteMessages submits a "Bridge.submit_bulk" call
```

Comment:
Comment is outdated, but doesn't matter if you're planning on adding `submit_bulk` back